packages/create-pantheon-decoupled-kit/src/actions/convertCSSModules.ts

```diff
@@ -39,18 +38,22 @@ export const convertCSSModules: Action = async ({ data }) => {
 			});
 			fs.writeFileSync(file, result);
 		}
+		const dirs = data.gatsby
+			? './src/**/*.jsx'
+			: './pages/**/*.jsx ./components/*.jsx';
 		execSync(
-			`npx css-modules-to-tailwind pages/**/*.jsx components/*.jsx --force`,
+			`npx --prefer-online --yes css-modules-to-tailwind ${dirs} --force`,
```

Comment:
@mitchellmarkoff This should make sure `npx` always fetches from the registry. Eventually we can abstract out a `runNpx` or `runPackageCommand` type helpers to make the `execSync` stuff a bit nicer